lib/fluent/plugin/formatter_tsv.rb

```diff
@@ -25,9 +25,14 @@ class TSVFormatter < Formatter
       config_param :keys, :array, value_type: :string
       desc 'The delimiter character (or string) of TSV values'
       config_param :delimiter, :string, default: "\t"
+      desc 'The parameter to enable writing to new lines'
+      config_param :add_newline, :bool, default: true
 
       def format(tag, time, record)
-        @keys.map{|k| record[k].to_s }.join(@delimiter)
+        formatted = ""
+        formatted << @keys.map{|k| record[k].to_s }.join(@delimiter)
```

Comment:
`formatted = @keys.map{|k| record[k].to_s }.join(@delimiter)` seems enough

Reply:
@repeatedly Thanks. I fix it.